src/entities/DefaultEntityBehavior/Validator.ts

```diff
@@ -24,6 +25,9 @@ const validator: EntityValidator = (
     const pattern: RegExp | undefined = (properties[idx] as ScalarProperty)
       ?.pattern;
     if (pattern && !(values[idx] + '').match(pattern)) {
+      if (values[idx] === '') {
```

Comment:
I guess that value could be null or undefined as well
```suggestion
      if (!values[idx]) {
```

Reply:
Appart from that, this should ignore the RegExp only if the property is not required